src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,107 @@
+package seedu.addressbook.ui;
+
+import seedu.addressbook.commands.CommandResult;
+
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.*;
+
+public class Formatter {
```

Comment:
Header comments for pubic classes. https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html Have a look at comments #2.